decidim-elections/app/controllers/decidim/elections/admin/questions_controller.rb

```diff
@@ -26,6 +26,11 @@ def create
               flash.now[:alert] = I18n.t("questions.create.invalid", scope: "decidim.elections.admin")
               render action: "new"
             end
+
+            on(:already_started) do
```

Comment:
The event key here is wrong because of which we get HTTP `204 No Content` when posting the form.

It should be:
```suggestion
            on(:election_started) do
```

Could you also please add a spec that tests this situation that when a new question is being added to an election that has already started, it would show this error message.

Otherwise this is really nice improvement and something I was also confused about when learning about the elections.

Reply:
Hi @ahukkanen -- I've done the change you requested. As for the spec, there's a spec for this already: https://github.com/decidim/decidim/pull/9189/files#diff-333ae91e97cc219e383da9e0377588748b34233d9948204ff773d6a176a81982R74

Reply:
Yes I noticed that @josepjaume but I meant to test the controller request or a system test for the create question to cover this issue. I had to test it manually to catch the bug, so it would be good to always add a regression test when we fix issues like this.

The linked spec is only covering the command and left that bug unnoticed.

Reply:
>  to test the controller request or a system test for the create question to cover this issue.

@josepjaume seems like that's still missing, can you add it please? Thanks!

Reply:
Hi! This is done - I just added a system spec to cover this.